libs/platform/src/lib/components/form/form-generator/dynamic-form-control-field.directive.ts

```diff
@@ -10,6 +10,27 @@ import { DynamicFormControl } from './dynamic-form-control';
     selector: '[fdpDynamicFormControlField]'
 })
 export class DynamicFormControlFieldDirective implements OnInit {
+    /**
+     * @description Form control instance.
+     */
+    @Input()
+    set fdpDynamicFormControlField(value: DynamicFormControl) {
```

Comment:
We dont need to prefix things with fdp except selectors. plus I would shorten this setter and getter. maybe just a `control=`

Reply:
@fkolar the name of this input is done in such manner in order to use this syntax: *fdpDynamicFormControlField="control; show: shouldShow". https://angular.io/guide/structural-directives

Reply:
This just the example. What I tried to point out some simplicity in naming:

with fdpXXX you are probably right if we want to follow this and we keep it the same across all platform - in other directives. 

1) Do we need to duplicate directory names?  `/form-generator/form-generator/`
2) Maybe for this we picked less intuitive name. Are we going to use these dynamic directive somewhere else? Probably only under this form-generator right? So maybe we dont need to use `Dynamic` everywhere. just keep it simple. 

3) I see JS docs says `Structural directive, which hides current form item and toggles validation rules for it.`, maybe we could call this little bit close to what it does ? Or jsut `FormControlField` would be enough


I am not just comfortable with long names, where we duplicate parent structure. It is the same as to say: 

FormGroupComponent 
    FormGroupFormFieldComponent
    FormGroupFormFieldControlComponent
    
    etc..

Reply:
Hi @fkolar 
In my opinion, it is refactoring and it is out of the scope of this issue, so I created new issue for it.
https://github.com/SAP/fundamental-ngx/issues/6381